webrtc-stats.html

```diff
@@ -1143,9 +1143,20 @@
              boolean                       readable;
              unsigned long long            bytesSent;
              unsigned long long            bytesReceived;
-             double                        roundTripTime;
+             double                        averageRtt;
+             double                        currentRtt;
```

Comment:
Well, the value of "Ta" (the time between connectivity checks) is up to the implementation. And it also depends on how many pairs you have and other factors. So it would be difficult to predict when connectivity checks are sent.

Consent checks, however, are recommended to be sent and random time intervals around 5 seconds, and the interval MUST be no shorter than 4 seconds.